src/logic/operate.js

```diff
@@ -8,9 +8,13 @@ const operate = (numberOne, numberTwo, operation) => {
       return firstNum.plus(secondNum).toString();
     case '-':
       return firstNum.minus(secondNum).toString();
-    case '/':
-      if (secondNum === '0') return '0';
-      return firstNum.div(secondNum).toString();
+    case '÷':
+      if (secondNum.toString() === '0') {
+        return '0';
```

Comment:
return an error message, not ZERO...someting like `"INFINITY"`

Reply:
Thanks @shloch you are right, I will make the change